src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -528,6 +528,12 @@ export class CategoryFacet extends Component {
 
     for (let i = 0; i < sortedParentValues.length; i++) {
       currentParentValue = currentParentValue.renderAsParent(sortedParentValues[i]);
+
+      // We do not want to make the "last" parent selectable, as clicking it would be a noop (re-selecting the same filter)
+      if (i != sortedParentValues.length - 1) {
```

Comment:
What about storing the condition in a variable called `isLastParent` and checking `if(!isLastParent)`?